pkg/controller/clusterdeployment/metrics.go

```diff
@@ -21,14 +21,14 @@ var (
 		prometheus.HistogramOpts{
 			Name:    "hive_cluster_deployment_install_job_duration_seconds",
 			Help:    "Distribution of the runtime of completed install jobs.",
-			Buckets: []float64{60, 300, 600, 1200, 1800, 2400, 3000, 3600},
+			Buckets: []float64{1800, 2400, 3000, 3600, 4500, 5400, 7200},
 		},
 	)
 	metricInstallDelaySeconds = prometheus.NewHistogram(
 		prometheus.HistogramOpts{
 			Name:    "hive_cluster_deployment_install_job_delay_seconds",
 			Help:    "Time between cluster deployment creation and creation of the job to install/provision the cluster.",
-			Buckets: []float64{30, 60, 120, 300, 600, 1200, 1800},
+			Buckets: []float64{60, 120, 180, 240, 300, 600, 1200, 1800, 2700, 3600},
```

Comment:
@newtonheath Even for the 99th percentile - we don't exceed 600. I see the point of adding more buckets for <600, but why do we need more upwards of 3 times the ceiling value?
![image](https://user-images.githubusercontent.com/54181650/167929525-79d908c6-0e3f-4fad-95be-271283f4a1f7.png)

Reply:
We've had over 30 minutes for hive while I have been looking, so I'd like double that, and I threw one in between. You must be missing some data as I see this which is 19 minutes, 1140 seconds

![Screen Shot 2022-05-11 at 4 27 08 PM](https://user-images.githubusercontent.com/91277/167941843-4cc7fc1c-048b-47ca-b4f8-5f8b575ef555.png)
.